CHANGES

```diff
@@ -4,7 +4,10 @@
 Notable User Facing Changes
 ---------------------------
 - Added support for Coarse-Grained buffer SVM on CUDA devices.
-
+- AlmaIF: Add DBDevice backend, which can be used to transparently
```

Comment:
We have moved the notes to doc/notes_5_0.rst. Can you move your addition there and leave here in CHANGES a mention that we have moved the change logs to doc/notes*.rst? You can write a bit more extensive description of the feature there as well to advertise the work, perhaps getting someone to test it.

Reply:
done